website/pages/en/_meta.js

```diff
@@ -42,6 +42,8 @@ export default {
   },
   'operating-graph-node': '',
   firehose: 'Firehose',
+  graphcast: 'Graphcast',
+  'mips-faqs': 'MIPs FAQs',
```

Comment:
Since you're not overriding the titles, better to set them to an empty string to use what's set in the frontmatter of the respective MDX files. That way, if it changes, we only have to update it in one place.
```suggestion
  graphcast: '',
  'mips-faqs': '',
```

Reply:
Ahh, makes sense. Thank you!